snapcraft/tests/commands/__init__.py

```diff
@@ -74,9 +74,11 @@ class CommandBaseTestCase(tests.TestCase):
     def setUp(self):
         super().setUp()
         self.runner = CliRunner()
+        self.obj = dict()
```

Comment:
it's so sad that click is using a name like obj. Even something like `user_data` would be better here. Oh well...